multi-class-classification/recipe.yaml

```diff
@@ -0,0 +1,69 @@
+# `recipe.yaml` is the main configuration file for an MLflow Recipe.
+# Required recipe parameters should be defined in this file with either concrete values or
+# variables such as {{ INGEST_DATA_LOCATION }}.
+
+# Variables must be dereferenced in a profile YAML file, located under `profiles/`.
+# See `profiles/local.yaml` for example usage. One may switch among profiles quickly by
+# providing a profile name such as `local` in the Recipe object constructor:
+# `p = Recipe(profile="local")`
+#
+# NOTE: YAML does not support tabs for indentation. Please use spaces and ensure that all YAML
+# files are properly formatted.
+
+recipe: "classification/v1"
+# Specifies the name of the column containing targets / labels for model training and evaluation
+target_col: "class"
+# Specifies the value of `target_col` which is considered the positive class: red wine in this case.
+#positive_class: 1
+# Sets the primary metric to use to evaluate model performance. This primary metric is used
+# to sort MLflow Runs corresponding to the recipe in the MLflow Tracking UI
+primary_metric: "f1_score_macro"
+steps:
+  ingest: {{INGEST_CONFIG}}
+  split:
+    # Train/validation/test split ratios
+    split_ratios: {{SPLIT_RATIOS|default([0.75, 0.125, 0.125])}}
+    # Specifies the method to use to perform additional cleaning on split datasets
+    # Note that arbitrary transformations should go into the transform step
+    # post_split_filter_method: create_dataset_filter
+  transform:
+    using: custom
+    # Specifies the method that defines the data transformations to apply during model inference
+    transformer_method: transformer_fn
+  train:
+    using: custom
+    # Specifies the method that defines the estimator type and parameters to use for model training
+    estimator_method: estimator_fn
+    tuning:
+        enabled: true
+        max_trials: 2
+        sample_fraction: 0.5
+        parameters:
+            alpha:
+                distribution: "uniform"
+                low: 0.0
+                high: 0.01
+            penalty:
+                values: ["l2", "l1"]
+            eta0:
+                distribution: "normal"
+                mu: 0.01
+                sigma: 0.0001
```

Comment:
We have 3 dimensions to tune yet only have 2 max_trials. This is inherently inconsistent. Please reduce the tuning dimension to 1 or at most 2, and increase the max_trials to actually do some tuning. Otherwise prefer to turn tuning off.